.github/workflows/autotests.yml

```diff
@@ -84,21 +98,21 @@ jobs:
             wheel: 'win_amd64'
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python ${{ env.PYTHON_V }}
       uses: actions/setup-python@v3
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: ${{ env.PYTHON_V }}
 
     - name: Fetch previously compiled Quandelibc
-      if: github.event_name == 'workflow_dispatch' && inputs.quandelibc_ref != ''
+      if: github.event_name == 'workflow_dispatch'
```

Comment:
revert and let the test != '', as the default value is '' and this will fail when using the default value